PD-1499 / 25.04 / Pd 1499 create do not create root dataset share warning (#3200)
truenas/documentation: static/includes/RootLevelDatasetShareWarning.md

&NewLine;

> {{< hint type=tip title="Do Not Create Pool-Level Dataset Shares" >}}

Reviewer: Wondering if this would be better as a yellow box than green, since it's more than just a tip.

```suggestion
{{< hint type=important title="Do Not Create Pool-Level Dataset Shares" >}}
```

> When creating a share, do not attempt to setup the root or pool-level dataset for the share.

Reviewer: ```suggestion
When creating a share, do not attempt to set up the root or pool-level dataset for the share.
```

Instead, create a new dataset under the pool-level dataset for the share.

> Setting up a share using the root dataset leads to storage reconfiguration issues.

Reviewer: ```suggestion
Setting up a share using the root dataset leads to storage configuration issues.
```
